packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -182,7 +192,15 @@ const sidePaneStyles: IStackStyles = {
   }
 };
 
-const availableSpaceStyles: IStackStyles = { root: { width: '100%', height: '100%' } };
+const availableSpaceStyles = (theme: Theme): IStackStyles => ({
+  root: {
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
```

Comment:
Just curious on the reason to overlay the chat on top of the call composite instead of setting the call composite visbility to hidden?

Reply:
Never came to my mind :D
Tested it and it works gr8 Haha

Reply:
Oo good to know!